composer.json

```diff
@@ -26,8 +26,8 @@
     "ext-curl": "*"
   },
   "require-dev": {
-    "phpunit/phpunit": "9.6.*",
-    "phpmailer/phpmailer": "6.8.*",
+    "phpunit/phpunit": "^9.6",
+    "phpmailer/phpmailer": "^6.8",
```

Comment:
Let's change this back to using wildcards. Eventually, we want to use wildcards for everything for clarity.